### PR TITLE
feature: Update member privileges documentation CY-3843

### PR DESCRIPTION
**Pull request by carlospinho04**

### CODE REVIEW

File: docs/organizations/roles-and-permissions-for-synced-organizations.md [hunk 1]
```diff
@@ -189,10 +206,10 @@ Depending on your role on the Git provider you will have different permissions o
 
 See [managing people](managing-people.md) to list and manage the members of your organization.
 
-## Configuring who can ignore issues {: id="configure-ignore-issues"}
+## Configuring who can ignore issues {: id="change-analysis-configuration"}
```
Comment (pauloribeiro-codacy) [inline docs/organizations/roles-and-permissions-for-synced-organizations.md:209]: We also need to update the title of the section:

```suggestion
## Configuring who can change analysis configurations {: id="change-analysis-configuration"}
```

File: docs/organizations/roles-and-permissions-for-synced-organizations.md [hunk 2]
```diff
@@ -189,10 +206,10 @@ Depending on your role on the Git provider you will have different permissions o
...
 
-By default, only users with Write permission on a repository can [ignore issues](../repositories/issues.md#ignoring-and-managing-issues), but you can change this. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to ignore issues for your organization.
+By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues for your organization](../repositories/issues.md#ignoring-and-managing-issues) and [configure your code patterns](../repositories-configure/code-patterns.md).
```
Comment (pauloribeiro-codacy) [inline docs/organizations/roles-and-permissions-for-synced-organizations.md:211]: Suggestion to clarify that both ignoring issues and configuring code patterns apply at the organization level:

```suggestion
By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues](../repositories/issues.md#ignoring-and-managing-issues) and [configure your code patterns](../repositories-configure/code-patterns.md) on your organization.
```

File: docs/repositories/issues.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ Use the options in the cogwheel menu of each issue to:
     See [how to restore ignored issues](#restoring-ignored-issues).
 
     !!! tip
-        Organization owners can [configure who is allowed to ignore issues](../organizations/roles-and-permissions-for-synced-organizations.md#configure-ignore-issues).
+        Organization owners can [configure who is allowed to ignore issues](../organizations/roles-and-permissions-for-synced-organizations.md#change-analysis-configuration).
```
Comment (pauloribeiro-codacy) [inline docs/repositories/issues.md:64]: Perhaps we can add an analogous "tip" card on the page [Code patterns](https://docs.codacy.com/repositories-configure/code-patterns/), at least temporarily to give more visibility to this new feature. :thinking:

**Comment by pedrocodacy** [comment]
On the image that we are replacing, can we have the select list open to show the allowed values like we had before?

Like this one (but with the copy change)
image

**Review by pauloribeiro-codacy — CHANGES_REQUESTED** [review]
@carlospinho04, I would also like to give feedback/tweak the copy for the new page, [as proposed here](https://docs.google.com/document/d/1Si31wtrv7Q4--Ml-pe2H259QRWtj44OMUkkD2yJk-fo/edit#heading=h.h9dud6h3e8ux):

* Change the title of the page to **Member privileges**
* Remove the sentence "Lower permission levels also grant access to the levels above", and mention the lower permission level directly before the drop-down list

Can you perform these changes or do you prefer that I open a pull request on codacy-website?

**Comment by carlospinho04** [comment]
> @carlospinho04, I would also like to give feedback/tweak the copy for the new page, [as proposed here](https://docs.google.com/document/d/1Si31wtrv7Q4--Ml-pe2H259QRWtj44OMUkkD2yJk-fo/edit#heading=h.h9dud6h3e8ux):
> 
> * Change the title of the page to **Member privileges**
> * Remove the sentence "Lower permission levels also grant access to the levels above", and mention the lower permission level directly before the drop-down list
> 
> Can you perform these changes or do you prefer that I open a pull request on codacy-website?

The new design is this one: 
image
Do you still want to remove the sentence "Lower permission levels also grant access to the levels above"?

**Comment by pauloribeiro-codacy** [comment]
> Do you still want to remove the sentence "Lower permission levels also grant access to the levels above"?

Yes, this was in fact a separate feedback ([CY-4012](https://codacy.atlassian.net/browse/CY-4012)), but since we're updating the page, if we do all the changes at once we can close both issues. :sunglasses:

**Comment by carlospinho04** [comment]
> > Do you still want to remove the sentence "Lower permission levels also grant access to the levels above"?
> 
> Yes, this was in fact a separate feedback ([CY-4012](https://codacy.atlassian.net/browse/CY-4012)), but since we're updating the page, if we do all the changes at once we can close both issues. 😎

Since this is still an open discussion and outside the scope of this epic, I would recommend not change the UI in this task

**Comment by pauloribeiro-codacy** [comment]
> > > Do you still want to remove the sentence "Lower permission levels also grant access to the levels above"?
> > 
> > 
> > Yes, this was in fact a separate feedback ([CY-4012](https://codacy.atlassian.net/browse/CY-4012)), but since we're updating the page, if we do all the changes at once we can close both issues. sunglasses
> 
> Since this is still an open discussion and outside the scope of this epic, I would recommend not change the UI in this task

OK, makes sense. I'll open a separate pull request. :+1:

File: docs/repositories/issues.md (hunk 3 above)
Comment (carlospinho04) [reply docs/repositories/issues.md:64]: Looks a bit out of scope, can you create an issue for this?

Comment (pauloribeiro-codacy) [reply docs/repositories/issues.md:64]: I would prefer to make this change in the same scope of this pull request, as it is a direct consequence of allowing organization owners to configure who can make changes to code patterns. I can make the change myself, though.

File: docs/organizations/roles-and-permissions-for-synced-organizations.md [hunk 4]
```diff
@@ -27,53 +28,59 @@ Depending on your role on the Git provider you will have different permissions o
...
+      <td><a href="#change-analysis-configuration">Configurable</a></td>
       <td>Yes</td>
       <td>No</td>
       <td>No</td>
     </tr>
     <tr>
       <td>Repository Admin</td>
       <td>Yes<sup><a href="#note-2">2</a></sup></td>
-      <td><a href="#configure-ignore-issues">Configurable</a></td>
+      <td>Yes<sup><a href="#note-2">2</a></sup></td>
```
Comment (pauloribeiro-codacy) [inline docs/organizations/roles-and-permissions-for-synced-organizations.md:80]: Good catch! :mag:

**Review by pauloribeiro-codacy — APPROVED** [review]

**Pull request merged**